app/ai/voice/agents/breeze_buddy/handlers/internal/load_data_source.py

```
@@ -0,0 +1,120 @@
+    try:
+        redis = await get_redis_service()
+        cached = await redis.get(cache_key)
+        if cached:
+            logger.info("[load_data_source] cache hit '%s' key=%s", source, key)
+            return {
+                "status": "success",
+                "source": source,
+                "key": key,
+                "content": cached,
+            }
+    except Exception as exc:  # never let cache issues break the call
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)